subworkflows/local/profiling.nf

```diff
@@ -362,7 +362,8 @@ workflow PROFILING {
         ch_input_for_krakenuniq =  ch_input_for_profiling.krakenuniq
             .map {
                 meta, reads, db_meta, db ->
-                    [[id: db_meta.db_name, single_end: meta.single_end], reads, db_meta, db]
+                    def seqtype = (reads[0].name ==~ /.+?\.f\w{0,3}a(\.gz)?$/) ? 'fasta' : 'fastq'
```

Comment:
One last doubt, if the input is single-end (as typical for FASTA), are we always converting the reads to a list, or might this be a single file and thus the index 0 might fail? I'm uncertain whether that is covered by test cases.

Reply:
In my testing it was always coming up as a list @Midnighter, I can triple check though but I don't see how it would change however if you prefer. For example I think in our tests profile sample sheet we only give a single FASTA, and we have a one single end data sample. Both generated expected (un) classified FASTQ files as expected